BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/GiniBankNetworkingScreenApiCoordinator.swift

```diff
@@ -233,6 +241,13 @@ open class GiniBankNetworkingScreenApiCoordinator: GiniScreenAPICoordinator, Gin
         })
         return self.start(withDocuments: documents, animated: animated)
     }
+
+    /// Attempts to cast the public-facing `transactionDocsDataCoordinator` to the internal protocol.
+    /// - Returns: The `TransactionDocsDataInternalProtocol` instance if the cast is successful, otherwise `nil`.
+    private func internalTransactionDocsDataCoordinator() -> TransactionDocsDataInternalProtocol? {
```

Comment:
[Suggestion] this method is not creating a new entity, it returns existing one with typecast. I suggest here to use computed property and remove it from inits of GiniBankNetworkingScreenApiCoordinator

```
/// Attempts to cast the public-facing `transactionDocsDataCoordinator` to the internal protocol.
/// - Returns: The `TransactionDocsDataInternalProtocol` instance if the cast is successful, otherwise `nil`.
private var internalTransactionDocsDataCoordinator: TransactionDocsDataInternalProtocol? {
    return GiniBankConfiguration.shared.transactionDocsDataCoordinator as? TransactionDocsDataInternalProtocol
}

```